conftest.py

```diff
@@ -420,7 +421,7 @@ def check_accuracy():
     """Fixture to check the accuracy."""
 
     def check_accuracy_impl(expected, actual, threshold=0.9):
-        accuracy = numpy.mean(expected == actual)
+        accuracy = accuracy_score(expected, actual)
```

Comment:
this is because we have https://github.com/zama-ai/concrete-ml-internal/issues/4029, which is already a fixme + skip in the modified tests below

basically we have a shape difference with sklearn for outputs for may models (we output (100,1) while sklearn gives (100,), so the previous formula was failing). Let's fix this is another PR !